src/ui/components/transactions/SetScript/SetScriptCardHeader.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as styles from './SetScriptCardHeader.module.css';
+import { useTranslation } from 'react-i18next';
+
+import { TxIcon } from '../BaseTransaction';
+
+interface IProps {
+  script?: string;
+}
+
+const SetScriptCardHeader: React.FC<IProps> = ({ script }) => {
```

Comment:
Is there any reason to use `React.FC` type? Function declaration like this would be enough:

```typescript
function SetScriptCardHeader({ script }: Props) {
```